### ISSUE TITLE
Non-responsive developer team for michael-jonas.github.io

### ISSUE THREAD

**Issue by LaurierMantel** [michael-jonas/michael-jonas.github.io#2]
In #1 I asked @michael-jonas to look into updating the website.  This hasn't happened, nor has the issue been acknowledged. 
I'm looking to start a discussion on:
- A potential [**RCA**](https://en.wikipedia.org/wiki/Root_cause_analysis) on why this has happened
- Best practices to preventing this in future work
- Ways to address the [growing backlog of issues](https://github.com/michael-jonas/michael-jonas.github.io/issues).

**Comment by LaurierMantel** [comment]
Still waiting on this @michael-jonas

**Comment by LaurierMantel** [comment]
ping @michael-jonas

**Comment by LaurierMantel** [comment]
Hey @michael-jonas any update on this? cc @ndey96 @rishigoel

**Comment by nolan-cerebras** [comment]
+1 @michael-jonas this is a major blocker for me as well

**Comment by rishigoel** [comment]
Same. Any clarity on this situation would be extremely valuable

**Comment by LaurierMantel** [comment]
@ndey96 can you follow up with the repo owner? Cc @rishigoel @michael-jonas
